### PR TITLE
Move from WindowsAzure.Storage library to Microsoft.AzureCosmos.Table and Microsoft.Azure.Storage.* packages.

### PR DESCRIPTION
**Pull request by sergeybykov**
This is to complete the work started by @seniorquico in #5479.

Marked as WIP for now because there are four functional test failures to investigate, likely test issues.

### CODE REVIEW

**Comment by seniorquico** [comment]
@sergeybykov Do you have the list of failing tests (when I click through to the Azure DevOps logs it shows 0 failures)? Additionally, are the instructions on configuring/running the functional test suite?

**Comment by sergeybykov** [comment]
Test issues have been sorted out. This is ready for review/merge.

**Comment by sergeybykov** [comment]
@seniorquico TestAll.cmd runs all functional tests. Some of them that have dependency on Azure Storage are skipped unless a OrleansTestSecrets.json with connection strings is found in the folder.
>{
  "DataConnectionString": "",
  "EventHubConnectionString":"
}

File: Directory.Build.props [hunk 1]
```diff
@@ -85,7 +85,10 @@
     <MicrosoftExtensionsHostingVersion>3.0.0</MicrosoftExtensionsHostingVersion>
 
     <MicrosoftApplicationInsightsVersion>2.4.0</MicrosoftApplicationInsightsVersion>
+    <MicrosoftAzureCosmosTableVersion>1.0.5</MicrosoftAzureCosmosTableVersion>
     <MicrosoftAzureEventHubsVersion>2.2.1</MicrosoftAzureEventHubsVersion>
+    <MicrosoftAzureStorageBlobVersion>10.0.3</MicrosoftAzureStorageBlobVersion>
+    <MicrosoftAzureStorageQueueVersion>10.0.3</MicrosoftAzureStorageQueueVersion>
```
Comment (ReubenBond) [inline Directory.Build.props]: Should we take the latest instead here? `11.1.0`

**Pull request merged**